styx-common/src/main/java/com/spotify/styx/storage/AggregateStorage.java

```diff
@@ -214,4 +214,9 @@ public void storeResource(Resource resource) throws IOException {
   public Optional<Backfill> backfill(String id) throws IOException {
     return datastoreStorage.getBackfill(id);
   }
+
+  @Override
+  public List<String> clientBlacklist() {
+    return datastoreStorage.clientBlacklist();
```

Comment:
Ok, I guess we could just deploy a new version of the server then that blacklists that client, but I don't have a very strong opinion.